airbyte-integrations/connectors/source-google-ads/integration_tests/test_incremental.py

```diff
@@ -58,7 +58,7 @@ def test_incremental_sync(config, configured_catalog):
             AirbyteLogger(),
             config,
             ConfiguredAirbyteCatalog.parse_obj(configured_catalog),
-            {"ad_group_ad_report": {"segments.date": latest_state}},
+            {"ad_group_ad_report": {config["customer_id"]: {"segments.date": latest_state}}},
```

Comment:
With a change to the state format, what will happen to existing incremental syncs when users upgrade?  Will they start over and perhaps see duplicate data?

Reply:
No, nothing out of the ordinary should happen. The connector supports old state format as well as the new one. Whenever it updates the state value, it is done according to the new format. This commit does not actually change state format (just moved logic from deprecated `get_updated_state` to `state` [property](https://github.com/airbytehq/airbyte/pull/12914/files#diff-3f54675c5be5786da7f7ad049d4216b6a2bd4490dd42e732615fbdfde12be1d5R149)). 

The change in tests is made because the connector emits exactly the same state message it got as an input whenever it is abnormally large, without updating the state format.